### ISSUE TITLE
Tests for Full Framework code

### ISSUE THREAD

**Issue by SergeyKleyman** [elastic/apm-agent-dotnet#200]
We agreed that it's would okay to merge https://github.com/elastic/apm-agent-dotnet/pull/195 PR 
without tests covering main added functionality and we will do it in a separate PR.

**Comment by SergeyKleyman** [comment]
@kuisathaverat and I discussed what we need from infrastructure point of view and agreed on the following

1. Automation team will implement a new step of CI pipeline that will bring up a clean temporary Windows+IIS machine (TBD whether it will be Docker or ephemeral VM, etc.). We also need to make sure built artifacts are available on that machine. (TBD whether we rebuilt everything on the temporary machine or use built artifacts from one of the permanent workers)
2. .NET agent team will implement test application, code to deploy it and  and start it on IIS and the tests . We will use xUnit.net as we do for the unit tests so that part of CI pipeline can be reused.
